grafana/risingwave-user-dashboard.dashboard.py

```diff
@@ -149,6 +152,18 @@ def section_overview(panels):
                     f"{metric('storage_stale_object_count')} >= bool 200",
                     "Lagging Vacuum",
                 ),
+                panels.target(
+                    f"{metric('state_store_meta_cache_size')} >= bool 2*1024*1024*1024*1.1",
+                    "Abnormal Meta Cache Memory",
+                ),
+                panels.target(
+                    f"{metric('state_store_block_cache_size')} >= bool 512*1024*1024*1.1",
+                    "Abnormal Block Cache Memory",
+                ),
+                panels.target(
+                    f"{metric('state_store_uploader_uploading_task_size')} >= bool 1024*1024*1024*0.7",
+                    "Abnormal Uploading Memory Usage",
+                ),
```

Comment:
The capacity of block cache, meta cache and uploading memory are configurable and will be set according to the node memory by default. See [here](https://github.com/risingwavelabs/risingwave/blob/572780bbca08c25a5d2f71016b915733e146df04/src/compute/src/memory_management/mod.rs#L119) for more details. Therefore, using a fix limit 2G, 512M and 1G for the alerting is not a good idea can can be misleading.

Reply:
I suggest we add kernel metrcis to emit usage ratio in addition to absolute value of the usage and we can use the ratio directly in the alerting.

Reply:
Make sense and fixed, thanks.